easybuild/easyconfigs/s/snpEff/snpEff-4.1d-Java-1.7.0_80.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'PackedBinary'
+
+name = 'snpEff'
+version = '4.1d'
+
+homepage = 'http://sourceforge.net/projects/snpeff/'
+description = """SnpEff is a variant annotation and effect prediction tool. 
+It annotates and predicts the effects of genetic variants (such as amino acid changes)."""
```

Comment:
ditto about 1 leading space in descriptions... (this relates to how a docstring ends up to become modulefile help)